specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,687 @@
+          {
+            "$ref": "#/parameters/policyName"
+          },
+          {
+            "$ref": "#/parameters/if-match"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicy"
+            },
+            "headers": {
+              "ETag": {
+                "description": "An HTTP entity tag associated with the policy entity",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy store service",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataPolicy_Unpublish": {
+            "$ref": "./examples/DataPolicy_UnPublish.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DataPolicyList": {
+      "required": [
+        "value"
+      ],
+      "description": "A paginated list of purview data policies.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "A list of purview data policies",
+          "items": {
+            "$ref": "#/definitions/DataPolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Pagination link"
+        }
+      }
+    },
+    "DataPolicy": {
+      "required": [
+        "name",
+        "decisionRules",
+        "members",
+        "description"
+      ],
+      "type": "object",
+      "description": "Purview data policy",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The policy name"
+        },
+        "state": {
+          "description": "The state of the policy",
+          "enum": [
+            "Published",
+            "Draft"
+          ],
+          "readOnly": true,
+          "type": "string",
+          "x-ms-enum": {
+            "name": "state",
+            "modelAsString": true
+          }
+        },
+        "description": {
+          "type": "string",
+          "description": "The policy description"
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The etag version of a policy"
+        },
+        "scopes": {
+          "type": "array",
+          "readOnly": true,
+          "description": "Array of scopes where the policy is published",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        },
```

Comment:
nit, What is a `scope`? Is it ARM concept? I think better doc would help here.

Reply:
updated the description.